app/controllers/admin/admin_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Admin::AdminController < ApplicationController
+  layout 'admin'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.